jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/BottomSheetTests.java

```diff
@@ -0,0 +1,49 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.bottomSheet;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class BottomSheetTests extends TestsInit {
+    private static final String[] BOTTOM_SHEET_VALUES = {"Google Keep", "Google Docs", "Google Plus", "Google Hangouts"};
+
+    @BeforeClass(alwaysRun = true)
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+        bottomSheet.show();
+    }
+
+    @Test
+    public void checkBottomSheetIsDisplayedTest() {
+        bottomSheet.open();
+        bottomSheet.is().isOpened();
+    }
+
+    @Test
+    public void checkBottomSheetIsHiddenTest() {
+        bottomSheet.open();
+        bottomSheet.is().isOpened();
+        bottomSheet.close();
+        bottomSheet.is().isClosed();
+    }
+
+    @Test
+    public void checkBottomSheetAvailableOptionsTest() {
+        bottomSheet.open();
+        bottomSheet.values().is().values(BOTTOM_SHEET_VALUES);
```

Comment:
bottomSheet.values().is().values(BOTTOM_SHEET_VALUES); -> bottomSheet.is().values(BOTTOM_SHEET_VALUES);